### PR TITLE
Added help how to use with GitLab CI

### PR DESCRIPTION
**Pull request by iBobik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,22 @@ docker run --rm -it \
...
 ```
 
+### GitLab CI
+
+```yaml
+html proofer:
+  stage: test
+  image:
+    name: klakegg/html-proofer:3.19.2
+    entrypoint: [""]
```
Comment (klakegg) [inline README.md:40]: Do we need to remove entrypoint just to add `htmlproofer` in the script section?

**Comment by iBobik** [comment]
Hi,
yes, GitLab requires shell-compatible entrypoint.


Honza Pobořil


29. 11. 2021 11:55:54, Erlend Klakegg Bergheim ***@***.***>
napsal:

**Comment by antoniovazquezblanco** [comment]
It would be awesome to get a `-ci` version of the images as in https://github.com/klakegg/docker-hugo/issues/13

Thanks!

**Comment by klakegg** [comment]
@iBobik Thank you.

@antoniovazquezblanco Less noise and more contributions, please.

**Pull request merged**